<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlSearchWidgetBase</class>
 <widget class="QWidget" name="BlSearchWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>77</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="mui_labelBusqueda">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string>BlSearchWidget</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="m_inputBusqueda">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="m_buscarWidget">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../bulmages.qrc">
       <normaloff>:/Images/edit-find.png</normaloff>:/Images/edit-find.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+F</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="m_clearSearchWidget">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Limpiar campo</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../bulmages.qrc">
       <normaloff>:/Images/edit-delete.png</normaloff>:/Images/edit-delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="m_textBusqueda">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <includes>
  <include location="local">local_BlI18n.h</include>
 </includes>
 <resources>
  <include location="../../bulmages.qrc"/>
 </resources>
 <connections/>
</ui>
